gdx/src/com/badlogic/gdx/graphics/Pixmap.java

```diff
@@ -127,28 +130,38 @@ public Pixmap (int width, int height, Format format) {
 	/** Creates a new Pixmap instance from the given encoded image data. The image can be encoded as JPEG, PNG or BMP.
 	 * @param encodedData the encoded image data
 	 * @param offset the offset
-	 * @param len the length */
-	public Pixmap (byte[] encodedData, int offset, int len) {
+	 * @param len the length
+	 * @param yUp load data y up */
+	public Pixmap (byte[] encodedData, int offset, int len, int yUp) {

```

Comment:
Please don't use `int` for what really is a `boolean`, at least in the public API, it is very confusing.